src/abaqus/Feature/Feature.py

```diff
@@ -61,7 +61,13 @@ class Feature:
     def AttachmentPoints(
         self,
         name: str,
-        points: float,
+        points: Union[
+            Sequence[ConstrainedSketchVertex],
+            Sequence[DatumPoint],
+            Sequence[MeshNode],
+            Sequence[InterestingPoint],
+            Sequence[Sequence[float]],
+        ],
```

Comment:
Use `|` instead of `Union`, and for all other changes

Reply:
Actually, I notice that the `Union` was used in some files like MeshPart.py, so I just revert the changes in Feature.py only, and still use `Union` in other files.